tests/boards/espressif/ethernet/prj.conf

```diff
@@ -0,0 +1,18 @@
+CONFIG_ZTEST=y
+
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_IPV6=n
+CONFIG_NET_IPV4=y
+CONFIG_NET_ARP=y
+CONFIG_NET_UDP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_DHCPV4_OPTION_CALLBACKS=y
+CONFIG_DNS_RESOLVER=y
+
+CONFIG_INIT_STACKS=y
+CONFIG_NET_MGMT=y
+CONFIG_NET_MGMT_EVENT=y
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
```

Comment:
until we get a proper handle for `SRAM_SIZE`, please add `CONFIG_SYS_HEAP_AUTO=y` in here to make sure heap will init successfully.